maven-plugin/src/test/java/org/openmrs/maven/plugins/utility/ArtifactHelperIT.java

```diff
@@ -0,0 +1,48 @@
+package org.openmrs.maven.plugins.utility;
+
+
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.maven.it.VerificationException;
+import org.junit.Test;
+import org.openmrs.maven.plugins.AbstractMavenIT;
+import org.openmrs.maven.plugins.model.Artifact;
+
+import java.io.File;
+
+import static org.junit.Assert.assertTrue;
+
+@Getter @Setter
+public class ArtifactHelperIT extends AbstractMavenIT {
+
+	@Test
+	public void test_downloadModuleWithDefaultName() throws Exception {
+		executeTest(() -> {
+			ArtifactHelper artifactHelper = new ArtifactHelper(getMavenEnvironment());
+			Artifact artifact = new Artifact("idgen-omod", "4.14.0", "org.openmrs.module", "jar");
+			artifactHelper.downloadArtifact(artifact, getMavenTestDirectory(), null);
+			File expectedFile = new File(getMavenTestDirectory(), "idgen-omod-4.14.0.jar");
+			assertTrue(expectedFile.exists());
+		});
+	}
```

Comment:
And so this is an example of how we would run such a test.  By wrapping the code in the `executeTest` block, it will cause the code in that block to be executed by a bare-bones plugin that exists only to execute the provided block.